hnn_core/tests/test_dipole.py

```diff
@@ -49,14 +50,40 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
                        " attribute 'append'"):
         plot_dipole(np.array([dipole, dipole]), average=True, show=False)
 
-    # Test IO
-    dipole.write(dpl_out_fname)
+    # Test IO for txt files
+    with pytest.warns(DeprecationWarning,
+                      match="Writing dipole to txt file is "
+                      "deprecated"):
+        dipole.write(dpl_out_fname)
     dipole_read = read_dipole(dpl_out_fname)
     assert_allclose(dipole_read.times, dipole.times, rtol=0, atol=0.00051)
     for dpl_key in dipole.data.keys():
         assert_allclose(dipole_read.data[dpl_key],
                         dipole.data[dpl_key], rtol=0, atol=0.000051)
 
+    # Test IO for hdf5 files
+    dipole.write(dpl_out_hdf5_fname)
+    dipole_read_hdf5 = read_dipole(dpl_out_hdf5_fname)
+    assert_allclose(dipole_read_hdf5.times, dipole.times, rtol=0, atol=0.00051)
+    for dpl_key in dipole.data.keys():
+        assert_allclose(dipole_read_hdf5.data[dpl_key],
+                        dipole.data[dpl_key], rtol=0, atol=0.000051)
```

Comment:
we should move away from using `atol` and use `rtol` ... it's more robust I believe

Reply:
https://stackoverflow.com/questions/65909842/what-is-rtol-for-in-numpys-allclose-function

Checks out! The dipole varies pretty wildly in magnitude between subthreshold and suprathreshold simulations, so considering the precision of `floats` gets worse with bigger numbers `rtol` seems particularly appropriate